src/gui/programupdater.cpp

```diff
@@ -89,7 +89,7 @@ void ProgramUpdater::checkForUpdates()
           this, SLOT(rssDownloadFinished(QNetworkReply*)));
   // Send the request
   QNetworkRequest request(RSS_URL);
-  request.setRawHeader("User-Agent", QString("qBittorrent/%1 ProgramUpdater (www.qbittorrent.org)").arg(VERSION).toLocal8Bit());
```

Comment:
Revert this. SourceForge.net should know that it is our programupdater, in case it observes too much traffic (due to a qbt bug).